File: server/Database/Battle.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Com.GitHub.GlowingPotato.IslandDecimation.Server.Database {
    public class Battle {
        [Key]
        public ulong Id {
            get;
            set;
        }

        public DateTime Time {
            get;
            set;
        }

        public ulong WinnerId {
            get;
            set;
        }

        [ForeignKey("WinnerId")]
        public User Winner {
            get;
            set;
        }
    }
}

File: client/src/main/java/com/github/glowingpotato/islanddecimation/battle/Battle.java
package com.github.glowingpotato.islanddecimation.battle;

public class Battle {
    private int id;
    private long time;
    private int winnerID;
    public Battle(int id, long time, int winnerID) {
	this.id = id;
	this.time = time;
	this.winnerID = winnerID;
    }
    public int getId() {
        return id;
    }
    public void setId(int id) {
        this.id = id;
    }
    public long getTime() {
        return time;
    }
    public void setTime(long time) {
        this.time = time;
    }
    public int getWinnerID() {
        return winnerID;
    }
    public void setWinnerID(int winnerID) {
        this.winnerID = winnerID;
    }
}
